README.md

```diff
@@ -329,6 +330,16 @@ importing. By default, only modules listed under `dependencies` and
 importDevDependencies: true
 ```
 
+### `dangleComma`
+
+By default, ImportJS will add trailing commas in named exports.
```

Comment:
Could you change this to `By default, ImportJS will add trailing commas when constructing import statements with multiple named imports. `?